ocaml/xapi/xapi_host.ml

```diff
@@ -1465,7 +1465,11 @@ let replace_host_certificate ~__context ~type' ~host
   let old_certs = Db_util.get_host_certs ~__context ~type' ~host in
   let new_cert = write_cert_fs () in
```

Comment:
This overwrites the existing certificate. What happens when there's a failure and the previous certificates needs to be reinstated?

Reply:
Maybe I should rename them to `*.bak` such that they would not be picked up by bundling but they would be around for recovery.